roles/kubernetes/kubeadm/tasks/update_cluster_info.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Load ca.crt
+  shell: set -o pipefail && cat "{{ kube_apiserver_client_cert }}" | base64 --wrap=0
```

Comment:
That could use `slurp` rather than shell. If you have unwanted wrapping just remove it with a filter.